packages/abi-gen-wrappers/package.json

```diff
@@ -15,11 +15,14 @@
         "lint": "tslint --format stylish --project .",
         "fix": "tslint --fix --format stylish --project .",
         "pre_build": "yarn generate_contract_wrappers",
-        "clean": "shx rm -rf lib wrappers",
-        "generate_contract_wrappers": "abi-gen --abis  ${npm_package_config_abis} --template ../../node_modules/@0x/abi-gen-templates/contract.handlebars --partials '../../node_modules/@0x/abi-gen-templates/partials/**/*.handlebars' --output src/generated-wrappers --backend ethers"
+        "clean": "shx rm -rf lib src/generated-wrappers",
+        "generate_contract_wrappers": "yarn gen_ts && yarn gen_py",
+        "gen_ts": "abi-gen --abis  ${npm_package_config_abis} --template ../../node_modules/@0x/abi-gen-templates/contract.handlebars --partials '../../node_modules/@0x/abi-gen-templates/partials/**/*.handlebars' --output src/generated-wrappers --backend ethers",
+        "gen_py": "abi-gen --abis  ${npm_package_config_abisForPython} --template ../../node_modules/@0x/abi-gen-templates/contract.py.handlebars --partials '../../node_modules/@0x/abi-gen-templates/partials.py/**/*.handlebars' --output src/generated-wrappers/py --backend ethers --language Python"
     },
     "config": {
-        "abis": "../contract-artifacts/artifacts/@(AssetProxyOwner|DutchAuction|DummyERC20Token|DummyERC721Token|ERC20Proxy|ERC20Token|ERC721Proxy|ERC721Token|Exchange|Forwarder|IAssetProxy|IValidator|IWallet|MultiAssetProxy|OrderValidator|WETH9|ZRXToken|Coordinator|CoordinatorRegistry|EthBalanceChecker).json"
+        "abis": "../contract-artifacts/artifacts/@(AssetProxyOwner|DutchAuction|DummyERC20Token|DummyERC721Token|ERC20Proxy|ERC20Token|ERC721Proxy|ERC721Token|Exchange|Forwarder|IAssetProxy|IValidator|IWallet|MultiAssetProxy|OrderValidator|WETH9|ZRXToken|Coordinator|CoordinatorRegistry|EthBalanceChecker).json",
+        "abisForPython": "../contract-artifacts/artifacts/ERC20Token.json"
```

Comment:
Python should have it's own package called `0x.py` that is going to call `abi-gen` and generate the Python wrappers. That package is going to become the canonical 0x Python package. Can we create that package and move this stuff over?

Reply:
Discussed elsewhere.  I'm going to create a new folder under `packages` for this functionality to live in.  The canonical 0x Python package will live over under `../../python-packages` (and be published to PyPI, not npmjs.com).  The npm package (folder under `../packages`, never to be published) will just be hosting the build scripts for generating the Python wrappers.